### ISSUE TITLE
Inclusion to gnome circle

### ISSUE THREAD

**Issue by jubalh** [dino/dino#980]
Not sure this is relevant but:
Maybe you are interested in adding dino to https://circle.gnome.org/. Might help in spreading the word and seems related.
Wonder if they will require a flatpak though..

**Comment by kirschner** [comment]
Yes, they require flatpak: ["Be available to install as a Flatpak"](https://gitlab.gnome.org/Teams/Circle/-/blob/master/README.md#who-can-apply). Seems the last information about the status there is https://github.com/dino/dino/issues/453

**Comment by Aman9das** [comment]
#453 is closed now.
